File: src/main/ngapp/src/app/scoreUpdate.ts
export class ScoreUpdate {
  private _playerUUID: string;
  private _score: number = 0;

  constructor() {

  }

  get playerUUID(): string {
    return this._playerUUID;
  }

  set playerUUID(value: string) {
    this._playerUUID = value;
  }

  get score(): number {
    return this._score;
  }

  set score(value: number) {
    this._score = value;
  }
}

File: src/main/java/game/actors/ScoreUpdate.java
package game.actors;

import requests.spark.websockets.objects.JSONWeb;
import requests.spark.websockets.objects.messages.mapping.WSData;
import requests.spark.websockets.objects.messages.mapping.WSDataReference;

import java.util.UUID;

public class ScoreUpdate extends JSONWeb {
    @WSDataReference(WSData.SCORE_UPDATE_SCORE)
    private Integer score = 0;
    @WSDataReference(WSData.SCORE_UPDATE_PLAYER_UUID)
    private UUID playerUUID;

    public ScoreUpdate(Integer score, UUID playerUUID) {
        this.score = score;
        this.playerUUID = playerUUID;
    }

    public Integer getScore() {
        return score;
    }

    public void setScore(Integer score) {
        this.score = score;
    }

    public UUID getPlayerUUID() {
        return playerUUID;
    }

    public void setPlayerUUID(UUID playerUUID) {
        this.playerUUID = playerUUID;
    }
}
